docs/plugins.md

```diff
@@ -229,26 +229,28 @@ defaultOptions: {
 A `util` module from Prettier core is considered a private API and is not meant to be consumed by plugins. Instead, the `util-shared` module provides the following limited set of utility functions for plugins:
 
 ```ts
+type Quote = "\"" | "'";
+type SkipOptions = { backwards?: boolean };
 getMaxContinuousCount(str: string, target: string): number;
 getStringWidth(text: string): number;
```

Comment:
```suggestion
function getStringWidth(text: string): number;
```

Can you do this on all the functions below? Then this code block becomes valid TypeScript, which means that Prettier will format it!

Reply:
Yes I had the same idea ^^

Reply:
Nice. You forgot to run Prettier on the file though ;)

Reply:
Oh... common
I used `Save without format` so the lines don't wrap 😂
Should I add `<!-- prettier-ignore -->` or should I format it?

Reply:
Yeah, maybe it makes sense to add prettier-ignore.

Reply:
Why avoid the wrapping? It'd look better than scrolling: https://deploy-preview-6788--prettier.netlify.com//docs/en/plugins.html#utility-functions

Reply:
Very opinionated
I (as a consumer) prefer to be able to do Side-Scroll in the preview window